.github/workflows/deploy-production.yml

```diff
@@ -0,0 +1,61 @@
+name: Deploy to Production
+
+on:
+  push:
```

Comment:
I don't think we want to deploy to prod on pushes. We probably just want to do the workflow_dispatch